### DISCUSSION TITLE
Obsidian-style query links in `export.json`

### DISCUSSION THREAD

**Discussion by dpaetzel** [srid/emanote#513] [Q&A]
When I fetch and parse `export.json`, each file's `links` field seems to only contain links to files that are hard-coded in the file. Am I missing something?

What I'd like is the `links` field to also include links that are part of the file due to being a result of a `query` code block.

E.g. if file `F.md` contains

````markdown
```query
path:./Ideas/*
```
````

then I'd like `links` to also include all files in `./Ideas/*` without having to explicitly mention them in `F.md`.

This would be helpful when automatedly building (partial) graphs/mindmaps from my Zettelkasten. For example, I'd like the graphs to link to the files in scratchpad folders like `Ideas` where I simply add Zettels without wanting to explicitly link to them from other Zettels. I'm generating these partial graphs e.g. to identify files that are not linked to by any other files (I wrote a [simplistic Python script](https://github.com/dpaetzel/ematools) which uses `networkx` to do so).

**Comment by srid** [comment]
Evaluating the queries is certainly we can do as part of the export. 

PR welcome as always; the change probably would go around here:

https://github.com/srid/emanote/blob/c15560890f0daa10cb5553c73a9ef5445d0886b4/emanote/src/Emanote/View/Export.hs#L67
